llvm/lib/Target/AMDGPU/AMDGPURegBankLegalizeHelper.cpp

```diff
@@ -352,6 +352,34 @@ void RegBankLegalizeHelper::widenLoad(MachineInstr &MI, LLT WideTy,
   MI.eraseFromParent();
 }
 
+void RegBankLegalizeHelper::widenMMO(MachineInstr &MI) {
+  MachineFunction &MF = B.getMF();
+  assert(MI.getNumMemOperands() == 1);
```

Comment:
Verifier already checks this, also cast to GAnyLoad would clean this up a little